ietf/settings.py

```diff
@@ -1301,3 +1301,5 @@ def skip_unreadable_post(record):
     CSRF_TRUSTED_ORIGINS += ['http://localhost:8000', 'http://127.0.0.1:8000', 'http://[::1]:8000']
     SESSION_COOKIE_SECURE = False
     SESSION_COOKIE_SAMESITE = 'Lax'
+
+PUBLISH_IPR_STATES = ['posted', 'removed', 'removed_objfalse']
```

Comment:
Minor tweak needed: Please move this above the `if not production` block. Just before is fine (unless you see some other puddle of ipr app related settings to attach it to).